lib/db.py

```diff
@@ -51,6 +51,7 @@ class Club(Base):
     school_id = Column(Integer, ForeignKey('school.id'))
     description = Column(String(1024), nullable=False)
     img_type = Column(String(100), nullable=False)
+    calendar_id = Column(String(100), nullable=False)
```

Comment:
those cal ids are long, make sure they aren't getting truncated in the database

Reply:
Would allowing a longer string solve the problem?